third_party/grpc-rust/protoc-rust-grpc/src/lib.rs

```diff
@@ -68,7 +67,7 @@ pub fn run(args: Args) -> Result<()> {
 
     let mut includes = args.includes;
     if includes.is_empty() {
-        static DOT_SLICE: &'static [&'static str] = &["."];
+        static DOT_SLICE: &[&str] = &["."];
```

Comment:
I did it due to this error: https://github.com/project-oak/oak/issues/510#issuecomment-579894252
that started appearing after I added a dependency (not a build-dependency) on a `protoc-rust-grpc`

Reply:
Ah interesting, so it seems that these changes were made by us to satisfy clippy. @blaxill do you remember why it was necessary to revert them?

Reply:
I just pulled from upstream, the important change was the less restrictive `Cargo.toml` 2.8 -> 2 change, so that it could generate 2.10 `protoc` files similar to the other generated proto files.

Reply:
Why did nothing break when these changes were undone though?

Reply:
It didn't get "clippy'ed" as a build/dev dependency, but since we are wrapping it it is a full dependency of `oak_utils` and so is clippy'ed now.